codespell_lib/data/dictionary.txt

```diff
@@ -23596,6 +23635,9 @@ thurrow->thorough
 thurver->further
 thw->the, thaw,
 thyat->that
+tich->thick, tick,
```

Comment:
Suggest titch too

Reply:
added + added `teach`

Reply:
Thanks for adding titch, teach seems a huge leap from tich to me, both phonetically and spelling wise.

Reply:
swapped `teach` for `stitch`